tools/db_bench_tool.cc

```diff
@@ -446,6 +446,26 @@ DEFINE_int32(num_multi_db, 0,
 DEFINE_double(compression_ratio, 0.5, "Arrange to generate values that shrink"
               " to this fraction of their original size after compression");
 
+DEFINE_double(
+    overwrite_probability, -1.0,
```

Comment:
Or should we let 0.0 mean "disable"?

Reply:
Oh right! Here skipping the code snippet (disable) and p=0.0 is strictly equivalent (unlike in other benchmarks) - good catch.